extensions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -223,24 +240,31 @@ func RestoreRKE1Snapshot(client *rancher.Client, clusterName string, snapshotRes
 		return err
 	}
 
-	updatedCluster := cluster
-
-	updatedCluster.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableControlplane = initialControlPlaneValue
-	updatedCluster.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableWorker = initialWorkerValue
-
-	_, err = client.Management.Cluster.Update(cluster, updatedCluster)
+	logrus.Infof("Restoring snapshot: %v", snapshotRestore.EtcdBackupID)
+	err = client.Management.Cluster.ActionRestoreFromEtcdBackup(cluster, snapshotRestore)
 	if err != nil {
 		return err
 	}
 
-	logrus.Infof("Restoring snapshot: %v", snapshotRestore.EtcdBackupID)
-	err = client.Management.Cluster.ActionRestoreFromEtcdBackup(cluster, snapshotRestore)
+	err = wait.PollUntilContextTimeout(context.TODO(), 1*time.Second, defaults.OneMinuteTimeout, true, func(ctx context.Context) (done bool, err error) {
+		clusterResp, err := client.Management.Cluster.ByID(cluster.ID)
+		if err != nil {
+			return false, nil
+		}
+
+		if clusterResp.State != active {
+			return true, nil
+		}
+
+		return false, nil
+	})
 	if err != nil {
 		return err
 	}
 
-	err = wait.Poll(1*time.Second, defaults.ThirtyMinuteTimeout, func() (bool, error) {
-		clusterResp, err := client.Management.Cluster.ByID(clusterID)
```

Comment:
this is why there's an extra line rm'd ; in the past we must have missed that this wasn't converted to the contextTimeout